meetings/tool/src/main/resources/Messages_eu.properties

```diff
@@ -0,0 +1,2 @@
+notification.subject=A new meeting \u0027{0}\u0027 has been published in the site \u0027{1}\u0027
```

Comment:
@pablosanroman would you mind contributing a translation? Thanks!

Reply:
Hi @mpellicer , here are the translations!
notification.subject= "{0}" bilera berria argitaratu da gune honetan: "{1}"
notification.content= Gonbidatuta zaude bilera honetan parte hartzeko: "{0}"